app/main.py

```diff
@@ -2,0 +1,20 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError
+
+    source_file, destination_file = parts[1], parts[2]
+
+    if source_file == destination_file:
+        return
+
+    if not os.path.exists(source_file):
+        raise FileNotFoundError
```

Comment:
The `FileNotFoundError` should include a message to indicate which file was not found. Consider adding a message like `raise FileNotFoundError(f'Source file {source_file} does not exist')`.